.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "20:00"
+      timezone: "Pacific/Honolulu"
+    open-pull-requests-limit: 10
```

Comment:
Since multiple branches are being defined, could you add `target-branch: "main"` to make it more clear? Thanks!

Reply:
Before I do that, could you also read this: [link](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#target-branch). Do you understand what they mean by "When you use this option, the settings for this package manager will no longer affect any pull requests raised for security updates."? Doesn't seem obvious to me.